modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -3726,6 +3726,14 @@
 		"newReference": "SitemapManager _sitemapManager",
 		"to": "_sitemapManager.addURLElement"
 	},
+	{
+		"classNames": [
+			"AccountEntryLocalService"
+		],
```

Comment:
@MicaelleSilvaa did you check if that pattern didn't happen also on `AccountEntryLocalServiceUtil`, `AccountEntryService` and `AccountEntryServiceUtil`? If happen, please and on this pattern and tests for it, thx